Mathlib/Data/Finsupp/Basic.lean

```diff
@@ -1495,23 +1495,26 @@ Throughout this section, some `Monoid` and `Semiring` arguments are specified wi
 `[]`. See note [implicit instance arguments].
 -/
 
-
+-- porting note: changed `AddMonoid` to `Zero`
 @[simp]
-theorem coe_smul [AddMonoid M] [DistribSMul R M] (b : R) (v : α →₀ M) : ⇑(b • v) = b • ⇑v :=
+theorem coe_smul [Zero M] [SMulZeroClass R M] (b : R) (v : α →₀ M) : ⇑(b • v) = b • ⇑v :=
   rfl
 #align finsupp.coe_smul Finsupp.coe_smul
 
-theorem smul_apply [AddMonoid M] [DistribSMul R M] (b : R) (v : α →₀ M) (a : α) :
+-- porting note: changed `AddMonoid` to `Zero`
```

Comment:
@ericrbg; rather than waiting for you to backport, I've added these porting notes. When you do the backport, you can record the forward-port by removing them.